monitoring_service/server.py

```diff
@@ -44,10 +44,11 @@ class MonitoringService(gevent.Greenlet):
     def __init__(
         self,
         private_key: str,
-        state_db: StateDB = None,
-        transport: Transport = None,
-        blockchain: BlockchainMonitor = None,
-        monitor_contract_address: str = None
+        state_db: StateDB,
+        transport: Transport,
+        blockchain: BlockchainMonitor,
+        monitor_contract_address: str,
```

Comment:
Not necessary for this PR, but this should be a proper `Address` type.

Reply:
Done in https://github.com/raiden-network/raiden-monitoring-service/pull/26/commits/8f9d4dab9ef007b760fad2e5eee75b4a3216b8f5.